lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -339,6 +339,11 @@ void clean_stale_enb_state(
   // Remove the S1 context for UEs associated with old eNB association
   hashtable_key_array_t* keys =
       hashtable_uint64_ts_get_keys(&stale_enb_association->ue_id_coll);
+  if (keys == NULL) {
```

Comment:
if (keys == NULL) then we should skip the for loop to clear the UEs, but still call s1ap_remove_enb()